app/src/main/java/com/example/weathertimeapp/mvp/view/WeatherDetailsView.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.weathertimeapp.mvp.view
+
+import com.example.weathertimeapp.activities.WeatherDetailsFragment
+import com.example.weathertimeapp.data.entities.Forecast
+import com.example.weathertimeapp.mvp.contracts.WeatherDetailsContract
+
+class WeatherDetailsView (fragment: WeatherDetailsFragment): WeatherDetailsContract.View{
```

Comment:
be careful with this.. you should have a weak reference for the fragment here.. just like in the activities

Reply:
You have to create a FragmentView(fragment) like the ActivityView(activity)

Reply:
forgot about the weak reference, thanks for the remainder, added to the MVP